apidoc/Titanium/UI/ListView.yml

```diff
@@ -1247,33 +1254,42 @@ methods:
         summary: |
             Dictionary with the properties `x`, `y`. Pass in `animated` `true` as a second dictionary parameter.
             The `x` and `y` coordinates reposition the top-left point of the scrollable region of the list view.
-            The `animated` property is optional and set to `false` by default. 
+            The `animated` property is optional and set to `false` by default.
         type: Dictionary
     since: 3.4.0
 
   - name: setMarker
     summary: Sets a reference item in the list view.
     description: |
         This method replaces previous reference items with the current one.
-        See [marker](Titanium.UI.ListView.marker) event for details. 
+        See [marker](Titanium.UI.ListView.marker) event for details.
     parameters:
       - name: markerProps
         summary: Dictionary to describe the reference item.
         type: ListViewMarkerProps
     platforms: [iphone, ipad, android]
     since: 3.2.0
-    
+
   - name: addMarker
     summary: Adds a reference item in the list view.
     description: |
-        See [marker](Titanium.UI.ListView.marker) event for details. 
+        See [marker](Titanium.UI.ListView.marker) event for details.
     parameters:
       - name: markerProps
         summary: Dictionary to describe the reference item.
         type: ListViewMarkerProps
     platforms: [iphone, ipad, android]
     since: 4.1.0
 
+    - name: getSelectedRows
```

Comment:
Ok. You should move this as a property, because the getters are generated automatically. Example:
```yml
    - name: selectedRows
      summary: returns the selected rows when AllowsMultipleSelectionDuringEditing is set to true.
      description: Returns an array of selected <Titanium.UI.ListItem> cells.
      platforms: [iphone, ipad]
      since: "6.2.0"
```